### PR TITLE
Changes to move the cennzx derived queries to api derives and refine t…

### PR DESCRIPTION
**Pull request by KarishmaBothara**
- Updated commit hash in docker compose to point to the latest node develop commit

- Removed all code from crml-cennzx-spot.. However the folders for crml-* still exist without any code.. will search on how to get rid of it...

- Refined the test and used cennzx over cennzxSpot as node is updated

### CODE REVIEW

File: packages/api/src/derives/cennzx/cennzxTrade.e2e.ts [hunk 1]
```diff
@@ -0,0 +1,500 @@
...
+            }
+          });
+      });
+    });
+  });
+
+  describe('Trading Operations', () => {
+    it('Can trade from asset to core for exact core asset amount', async done => {
+      const recipient = null; // in which case the caller receives the exchanged amount
+      const assetToSale = assetA;
```
Comment (holygits) [inline packages/api/src/derives/cennzx/cennzxTrade.e2e.ts:148]: nit: `assetToSale` -> `assetToSell`/`assetForSale` need change in many other places also

File: packages/api/src/derives/cennzx/cennzxTrade.e2e.ts [hunk 2]
```diff
@@ -0,0 +1,500 @@
...
+      const assetToSell = assetA;
+      const assetToBuy = coreAssetId;
+      const sellAmount = 50;
+      const expectedCorePrice = await api.rpc.cennzx.sellPrice(assetToSell, sellAmount, assetToBuy);
+      const minSale = 1;
+      await api.tx.cennzx
+        .sellAsset(recipient, assetToSell, assetToBuy, sellAmount, minSale)
+        .signAndSend(alice, async ({ events, status }) => {
+          if (status.isFinalized && events !== undefined) {
+            let trade = false;
```
Comment (holygits) [inline packages/api/src/derives/cennzx/cennzxTrade.e2e.ts:257]: what's the purpose of this variable `let trade`?

**Review by holygits — COMMENTED** [review]
lvgtm.
some small tidy ups suggested in tests.

future improvement: I think
`tx.sellAsset` and `tx.buyAsset` would benefit alot from destructor syntax / typescript to help with which order the variables should go in.

File: packages/api/src/derives/cennzx/assetWithdrawn.ts [hunk 3]
```diff
@@ -18,7 +18,7 @@ import { Hash } from '@cennznet/types/interfaces';
 import BN from 'bn.js';
 import { combineLatest, Observable } from 'rxjs';
 import { map } from 'rxjs/operators';
-import { getAssetToWithdraw } from '../utils/utils';
+import { getAssetToWithdraw } from './utils/utils';
```
[the remark this answers is not in the document]
Comment (KarishmaBothara) [reply packages/api/src/derives/cennzx/assetWithdrawn.ts:21]: will change.. I just moved from the crml.. didn't realise..

File: packages/api/src/derives/cennzx/cennzxTrade.e2e.ts (hunk 2 above)
Comment (KarishmaBothara) [reply packages/api/src/derives/cennzx/cennzxTrade.e2e.ts:257]: don't need.. vll remove..

**Review by holygits — APPROVED** [review]

**Pull request merged**